docs/support/tutorials/gis/eo_guide.md

```diff
@@ -39,221 +57,176 @@ What to consider when chosing data:
+    
+[**NASA Earthdata**](https://search.earthdata.nasa.gov) provides among many others [harmonized Landsat 8 and Sentinel-2 dataset](https://hls.gsfc.nasa.gov/). It requires free registration and download is possible via web interface and bulk download.
 
-## How can I process the data?
+**[Amazon Web Service (AWS) open EO data](https://registry.opendata.aws/?search=tags:gis,earth%20observation,events,mapping,meteorological,environmental,transportation)** is a collection of worldwide EO datasets provided by different organizations, including Landsat and Sentinel. Some of the data can be downloaded only on "requestor pays" basis. Currently, [Sentinel-2 L2A Cloud-optimized Geotiffs](https://registry.opendata.aws/sentinel-2-l2a-cogs/) are available for free, also via STAC.
 
-At CSC, EO data can be processed and analyzed using for example [CSC's supercomputer Puhti](../../../computing/systems-puhti.md) or a virtual machine in the [CSC cloud service cPouta](../../../cloud/pouta/pouta-what-is.md). You can find more information around geocomputing using CSC resources on [CSC geocomputing pages](https://research.csc.fi/geocomputing).
+**[Microsoft planetary computer](https://planetarycomputer.microsoft.com)** provides a STAC of all available data, which includes Sentinel, Landsat, MODIS. It is currently available in preview.
 
-Puhti has a lot of applications already installed (see below), you do not need to worry about it. If you need further applications, you can ask us to install them for you. You can also add your own installations using for example the [CSC Tykky tool](../../../computing/containers/tykky.md).   In cPouta, you need to set up your own virtual machine including all security and software setup, see [CSC cPouta instructions](../../../cloud/pouta/launch-vm-from-web-gui.md).
+[**Google Cloud Storage open EO data**](https://cloud.google.com/storage/docs/public-datasets), including Sentinel-2 L1C and Landsat Collection 1 data. Data can be downloaded for example with [FORCE](../../../apps/force.md).
+    
+[**Terramonitor**](https://www.terramonitor.com/services/analysis-ready) provides pre-prosessed, analysis ready Sentinel-2 data from Finland available between 2018-2020. It is a commercial service.
 
-!!! default "Please note"
+!!! default "Other geospatial datasets"
 
-    Puhti (and also CSC's other supercomputers Mahti and LUMI), work differently from your own computer. Before starting to use them, please familiarize yourself with the environment by for example visit the respective chapters in our [CSC computing environment self-learning course](https://ssl.eventilla.com/csccompenvselflearn).
+    To find other geospatial datasets, check out [CSC open spatial dataset list](https://research.csc.fi/open-gis-data).  
 
-What to consider when chosing software:
 
+## How can I process EO data at CSC?
 * User skills and preferences
```

Comment:
The suggest change does not work here.. I would move this below the text with geocomputing page link. And change to this:

### What to consider when chosing software?

* Functionality 
* Interaction type:
    * Graphical User Interface (GUI)
    * Command Line Interface (CLI)
    * Scripting
* Technical aspects:
    * Support for parallel computing or batch processing
    * Computationally efficient 
    * Supported operating systems
    * Automation
    * Reproducibility
* Open source vs proprietary
    * Proprietary tools need licenses which may be expensive and/or limiting the use of the tool
    * FOSS (free and open source software) allows the user to inspect the source code and provide high level insights in its functionality

Reply:
added it into text, see last commit